src/databricks/labs/ucx/recon/workflows.py

```diff
@@ -0,0 +1,15 @@
+from databricks.labs.ucx.contexts.workflow_task import RuntimeContext
+from databricks.labs.ucx.framework.tasks import Workflow, job_task
+
+
+class MigrationRecon(Workflow):
+    def __init__(self):
+        super().__init__('recon-migration-result')
+
+    @job_task(job_cluster="main")
+    def recon_migration_result(self, ctx: RuntimeContext):
+        """This workflow validate post-migration datasets against their pre-migration counterparts. This includes all
+        tables, by comparing their schema, row counts and row comparison
+        """
+        ctx.migration_recon.reset()
```

Comment:
is the reset on an index or something else?

Reply:
need to delete the existing content of recon results table, so that snapshot will re-populate it